### PR TITLE
javadoc 

### PR DESCRIPTION
**Pull request by DorianRec**
javadoc added

### CODE REVIEW

**Review by 1-alex98 — DISMISSED** [review]

File: src/main/java/com/treasure/hunt/analysis/Statistic.java [hunk 1]
```diff
@@ -62,10 +76,17 @@ public double getSolutionQuotient() {
         return getLocalOptimumSolution() / getOptimumSolution();
     }
 
-    public double getHintRequests() {
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/analysis/Statistic.java]: replaces more than javadoc, merge at own risk

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java [hunk 2]
```diff
@@ -36,10 +33,9 @@
  * Bounding Area:         the biggest Area in which the Strategy wants to place the treasure
  * checkedArea:            the Area the player has visited and thus must not contain the target
  * AreaExcludedByHints:    the Area the previous hints have
- *
+ * <p>
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: where's the corresponding closing tag?

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java [hunk 3]
```diff
@@ -178,10 +176,10 @@ private void adaptBoundingCircle() {
 
     /**
      * Updates the GameField's state with a new Players SearchPath
-     *
-     *  this method assumes that the the searcher cannot fly, e.g. For 2 visited points,
-     *  all points on the line between them are counted as checked,
-     *  and of those point, all points within a distance of searcherScoutRadius are checked as well
+     * <p>
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: again: closing?

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java [hunk 4]
```diff
@@ -216,15 +214,15 @@ public void commitPlayerMovement(SearchPath searchPath) {
     }
 
     /**
-      * Integrates the 2 intersections between the bounding circle and the current hint,
-      * then merges the resulting polygon and the remaining possible area
-      * to the new possible area.
+     * Integrates the 2 intersections between the bounding circle and the current hint,
+     * then merges the resulting polygon and the remaining possible area
+     * to the new possible area.
+     * <p>
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java:220]: If you want to separate two paragraphs i suggest the `<br>` tag

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java [hunk 5]
```diff
@@ -334,40 +332,37 @@ public boolean isWithinGameField(Point p){
      */
     public List<Pair<Coordinate, Double>> getWorstPointsOnAllEdges(double minDistance) {
 
-
         // making the decision transparent by drawing all lines which are inspected
         Geometry innerBufferedArea = possibleArea.getObject().buffer(-0.001);
 
         // due to rounding errors in the calculation a minimally negative buffer is to be drawn
         //innerBufferItem = new GeometryItem(innerBufferedArea.copy(), GeometryType.INNER_BUFFER, innerBufferStyle);
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: inline comments (@axel1200 screeching)

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java [hunk 6]
```diff
@@ -334,40 +332,37 @@ public boolean isWithinGameField(Point p){
...
 
         Geometry geometryToCheck;
-        if(minDistance > this.searcherScoutRadius){
+        if (minDistance > this.searcherScoutRadius) {
             Circle minDistanceCircle = new Circle(startingPoint.getCoordinate(), minDistance, geometryFactory);
             geometryToCheck = innerBufferedArea.difference(minDistanceCircle);
-        }else{ //dont bother
+        } else { //dont bother
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: @axel1200 ! calm down! CALM DOWN! Don't you see that you hurt him?!

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java [hunk 7]
```diff
@@ -413,19 +408,21 @@ public int compare(Pair<Coordinate, Double> coordinateDoublePair, Pair<Coordinat
...
-        }else{
+        } else {
             return worstEdgePoints;
         }
     }
 
 
-    /**Todo
+    /**
+     * Todo
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: No

Comment (jotoh98) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: Big nono

Comment (jotoh98) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: The author of this file should add correct documentation. @Trampeltier @rukemn @DorianRec ?

File: src/main/java/com/treasure/hunt/strategy/hider/impl/HalfPlaneAdapter_FixedTreasure.java [hunk 8]
```diff
@@ -10,7 +9,7 @@
 
 /**
  * Adapter such that searchers who require HalfPlaneHints can play against this strategy
- *
+ * <p>
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/HalfPlaneAdapter_FixedTreasure.java]: closing tag

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java [hunk 9]
```diff
@@ -472,7 +469,6 @@ public int compare(Pair<Coordinate, Double> coordinateDoublePair, Pair<Coordinat
 
         LineSegment normalizedLineSegment = new LineSegment(p1Transformed, p2Transformed);
 
-
         //now normal state is reached and Derivative can be applied for MaxConstant Search
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: There are multiple inline comments, which I've not found. Please replace and write self-explaining methods. And If you want to explain it nevertheless (which you should) add javadoc to the method

File: src/main/java/com/treasure/hunt/strategy/hider/impl/StatisticalHider.java [hunk 10]
```diff
@@ -70,7 +69,7 @@ public AngleHint move(SearchPath searchPath) {
         gameField.commitPlayerMovement(searchPath);
 
         // generate Hints
-        List<AngleHint> possibleHints = generateHints(360, searchPath.getLastPoint() );
+        List<AngleHint> possibleHints = generateHints(360, searchPath.getLastPoint());
```
Comment (jotoh98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/StatisticalHider.java]: The more you know...

**Review by jotoh98 — DISMISSED** [review]

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java (hunk 2 above)
Comment (DorianRec) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: sorry, this was build by autoformatting

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java (hunk 3 above)
Comment (DorianRec) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: sorry, this was build by autoformatting

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java (hunk 4 above)
Comment (DorianRec) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java:220]: also generated by autoformatting

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java (hunk 7 above)
Comment (DorianRec) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: this were also build by autoformatting ..
I agree, but Im not the author ^^

File: src/main/java/com/treasure/hunt/strategy/hider/impl/HalfPlaneAdapter_FixedTreasure.java (hunk 8 above)
Comment (DorianRec) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/HalfPlaneAdapter_FixedTreasure.java]: again, generated by autoformatting

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java (hunk 5 above)
Comment (DorianRec) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: sorry, I dont wrote this ^^ but I could add a PR, deleting inline comments

File: src/main/java/com/treasure/hunt/analysis/Statistic.java (hunk 1 above)
Comment (DorianRec) [reply src/main/java/com/treasure/hunt/analysis/Statistic.java]: youre right

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java [hunk 11]
```diff
@@ -421,10 +421,9 @@ public int compare(Pair<Coordinate, Double> coordinateDoublePair, Pair<Coordinat
         }
     }
 
+    // TODO
```
Comment (1-alex98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: why?

File: src/main/java/com/treasure/hunt/strategy/hider/impl/HintEvaluator.java [hunk 12]
```diff
@@ -52,9 +51,8 @@ public void registerHint(AngleHint hint, double areaAfter) throws InvalidHintExc
             throw new InvalidHintException("given Hint does not have the player as Center");
         }
 
-
         //AngleHintStatistic ahs = new AngleHintStatistic(hint, currentArea, areaAfter);
-       // hints.add(ahs);
+        // hints.add(ahs);
```
Comment (1-alex98) [inline src/main/java/com/treasure/hunt/strategy/hider/impl/HintEvaluator.java]: No code comments delete those

**Review by 1-alex98 — APPROVED** [review]

**Review by 1-alex98 — DISMISSED** [review]
see above

File: src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java (hunk 11 above)
Comment (1-alex98) [reply src/main/java/com/treasure/hunt/strategy/hider/impl/GameField.java]: this is not resolved yet

**Review by 1-alex98 — DISMISSED** [review]

**Review by jotoh98 — DISMISSED** [review]

**Review by jotoh98 — APPROVED** [review]

**Pull request merged**